examples/3d_columns.rs

```diff
@@ -22,10 +25,10 @@ pub fn main() {
         .add_plugins(DefaultPlugins)
         .add_startup_system(setup_camera)
         .add_startup_system(setup_grid)
-        .add_system_set(
-            SystemSet::new()
-                .with_run_criteria(FixedTimestep::step(0.1))
-                .with_system(animate_rings),
+        .add_system(
+            animate_rings
+                .in_schedule(CoreSchedule::FixedUpdate)
```

Comment:
@alice-i-cecile is this line necessary ? It works without it, but the [docs](https://docs.rs/bevy/latest/bevy/time/common_conditions/fn.on_fixed_timer.html) suggest its required

Reply:
Hmm. Given that this is UI-based, I'd actually leave it out of `FixedUpdate`, and instead consider using `on_timer` run condition.